app/services/flow_content_item_retriever.rb

```diff
@@ -1,8 +1,7 @@
-class ContentItemRetriever
+class FlowContentItemRetriever
   def self.fetch(slug)
-    Services.content_store.content_item("/#{slug}")
+    Services.content_store.content_item("/#{slug}/y")
```

Comment:
I wonder if we have a path we can use here rather than having to create the path ourselves (and thus hardcode the /y)

Reply:
Although we do have access to the path, I'm not sure about if we'll hit issues around whether the format is set: https://github.com/alphagov/smart-answers/blob/master/config/routes.rb#L12

What do you think about instead providing a method which turns a slug into this path and then we can use it here as well:

https://github.com/alphagov/smart-answers/blob/f3790fb4650283827d1a0ab66bb5b21d85dabcc1/app/presenters/flow_content_item.rb#L38-L40